en/docs/deploy/work-with-properties-of-user-stores.md

```diff
@@ -15,22 +15,22 @@ to configure primary userstores.
 <tr class="odd">
 <td><code>             MaxUserName<br>ListLength            </code></td>
 <td><code>             max_user_name<br>_list_length        </code></td>
-<td>Controls the number of users listed in the userstore of a WSO2 product. This is useful when you have a large number of users and don't want to list them all. Setting this property to 0 displays all users.</td>
+<td>This controls the number of users listed in the userstore of WSO2 Identity Server. This is useful when you have a large number of users and don't want to list them all. Setting this property to 0 displays all users.</td>
 </tr>
 <tr class="even">
 <td><code>             Connection<br>URL            </code></td>
 <td><code>             connection<br>_url           </code></td>
-<td><p>Connection URL to the userstore server. In the case of default LDAP in Carbon, the port is specified in the <code>              carbon.xml             </code> file, and a reference to that port is included in this configuration.</p></td>
+<td><p>This is thee connection URL to the userstore server. In the case of default LDAP in Carbon, the port is specified in the <code> <&ZeroWidthSpace;IS_HOME>/repository/conf/carbon.xml </code> file, and a reference to that port is included in this configuration.</p></td>
 </tr>
 <tr class="odd">
 <td><code>             Connection<br>Name            </code></td>
 <td><code>             connection<br>_name           </code></td>
-<td><p>The username used to connect to the database and perform various operations. This user does not have to be an administrator in the userstore or have an administrator role in the WSO2 product that you are using, but this user MUST have permissions to read the user list and users' attributes and to perform search operations on the userstore. The value you specify is used as the DN ( <code>              Distinguish Name             </code> ) attribute of the user. This property is mandatory.</p></td>
+<td><p>This is the username used to connect to the database and perform various operations. This user does not have to be an administrator in the userstore or have an administrator role in WSO2 Identity Server, but this user needs to have permissions to read the user list and users' attributes, and to perform search operations on the userstore. The value you specify is used as the DN ( <code>              Distinguish Name             </code> ) attribute of the user. This property is mandatory.</p></td>
```

Comment:
`<code>              Distinguish Name             </code>`
is this required to be enclosed within code tags?